src/internal/serde/yaml_decoder.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/gogo/protobuf/jsonpb"
 	"github.com/gogo/protobuf/proto"
 	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
-	"gopkg.in/pachyderm/yaml.v3"
```

Comment:
We had forked `gopkg.in/yaml.v3` and modified it to parse adjacent json documents:
```
{
  ...
}{
  ...
}
```
in keeping with `encoding/json`. Again, though, since we're about to GA 2.0 it probably makes sense to move off our fork and just ask users to tolerate this change by adding three dashes between their pipelines specs:
```
{ ... }
---
{ ... }
```